### PR TITLE
Yongbing chen/receipt predicting

### PR DESCRIPTION
**Pull request by yongbing-chen**

### CODE REVIEW

File: src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx [hunk 1]
```diff
@@ -0,0 +1,989 @@
+// Copyright (c) Microsoft Corporation.
```
Comment (stew-ro) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx:1]: This component seems to contain lots of duplicate code from the analyze page. Lets remove duplicate code

File: src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
+// Copyright (c) Microsoft Corporation.
```
Comment (stew-ro) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx:1]: Also seems to have duplicate code

File: src/react/components/pages/prebuiltPredict/prebuiltPredictResult.scss [hunk 3]
```diff
@@ -0,0 +1,182 @@
+@import "../../../../index.scss";
```
Comment (stew-ro) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictResult.scss:1]: Also seems to have duplicate code

File: src/react/components/pages/prebuiltPredict/prebuiltPredictPage.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
+@import "../../../../assets/sass/theme.scss";
```
Comment (stew-ro) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictPage.scss:1]: Also seems to have duplicate code

File: src/react/components/pages/prebuiltPredict/filePicker.tsx [hunk 5]
```diff
@@ -0,0 +1,259 @@
+import {Dropdown, IDropdownOption, PrimaryButton, TextField} from '@fluentui/react';
```
Comment (stew-ro) [inline src/react/components/pages/prebuiltPredict/filePicker.tsx:1]: This is mostly duplciate code from the analyze page. Can we refactor?

Comment (stew-ro) [reply src/react/components/pages/prebuiltPredict/filePicker.tsx:1]: It's not specific to prebuilt page

**Comment by alex-krasn** [comment]
Misspelled  "prebuilt" in the header: 
image

**Comment by alex-krasn** [comment]
Misleading "Over the rate limitation...".
I don't think it's receipt PR specific, but it's exposed in this PR:
image

**Comment by yongbing-chen** [comment]
> Misleading "Over the rate limitation...".
> I don't think it's receipt PR specific, but it's exposed in this PR:
> image

fixed, please help review the code.

**Comment by yongbing-chen** [comment]
> Misspelled "prebuilt" in the header:
> image

fixed

File: src/react/components/pages/prebuiltPredict/prebuiltPredictResult.scss (hunk 3 above)
Comment (stew-ro) [reply src/react/components/pages/prebuiltPredict/prebuiltPredictResult.scss:1]: This is mostly a copy of predictResults.scss. I think we'll want to refactor out duplicate code. This way, we don't need to make changes in multiple files when we need to change something in common

File: src/react/components/pages/editorPage/canvasCommandBar.tsx [hunk 6]
```diff
@@ -46,69 +49,76 @@ export const CanvasCommandBar: React.FunctionComponent<ICanvasCommandBarProps> =
     }
 
     let commandBarItems: ICommandBarItemProps[] = [];
-    if (props.parentPage === strings.editorPage.title) {
-        commandBarItems = [{
+    // if (props.parentPage === strings.editorPage.title) {
```
Comment (stew-ro) [inline src/react/components/pages/editorPage/canvasCommandBar.tsx:52]: remove comments

File: src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx (hunk 2 above)
Comment (stew-ro) [reply src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx:1]: Can we factor out common code of the components? https://reactjs.org/docs/composition-vs-inheritance.html

File: src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+                {postProcessedValue &&
+                    <li className="predictiontag-item-label mb-1">
+                        {postProcessedValue}
+                    </li>
+                }
+            </div>
+        );
+    }
+
+    private getTagColor = (name: string): string => {
```
Comment (stew-ro) [inline src/react/components/pages/prebuiltPredict/prebuiltPredictResult.tsx:95]: An example of duplicate code that we want to refactor out. This is the same function in existing predictResult.tsx class

**Comment by alex-krasn** [comment]
> > Misspelled "prebuilt" in the header:
> > image
> 
> fixed
It hasn't been fixed - I pushed the change

**Comment by alex-krasn** [comment]
adde margin for results 
image ===> 
image

**Comment by alex-krasn** [comment]
renamed "Pre-built" to "Form type", and "BusinessCard" to "Business card" :
image

**Comment by alex-krasn** [comment]
image
 - Does not working

File: src/App.scss [hunk 8]
```diff
@@ -140,6 +140,28 @@
...
+}
+.icon-6x {
+    font-size: 6em;
+}
+.icon-7x {
+    font-size: 7em;
+}
+.icon-8x {
+    font-size: 8em;
+}
```
Comment (alex-krasn) [inline src/App.scss:164]: those classes been added, but never used

File: src/common/localization/en-us.ts [hunk 9]
```diff
@@ -57,6 +58,22 @@ export const english: IAppStrings = {
         messages: {
             deleteSuccess: "Successfully deleted ${project.name}",
         },
+        homeProjectView: {
+            title: "Train and use a model with labels"
+        },
+        prebuiltPredict: {
+            title: "Use prebuilt model to get data",
+            description: "Start with a pre-built model for forms link receipts or business cards. Submit your data and get result right away."
```
Comment (alex-krasn) [inline src/common/localization/en-us.ts:66]: We need to be consistent in wording and use either "pre-built" or "prebuilt", but not both everywhere.

**Comment by alex-krasn** [comment]
For home page styles we should keep home page tiles sizes the same (same height and width). Als, I think, the "Quick start guide" link should be inside the tile, just make sure its accessible/clickable.
image

**Review by XinxingLiu — DISMISSED** [review]

**Review by stew-ro — DISMISSED** [review]

**Review by alex-krasn — DISMISSED** [review]

**Comment by alex-krasn** [comment]
Also instead of adding margin to the cards, could add different value of ```$lighter-``` on hover - so it does not blends with sidebar on the left and this save screen space.

**Review by starain-pactera — APPROVED** [review]

File: src/App.scss (hunk 8 above)
Comment (yongbing-chen) [reply src/App.scss:164]: icon-7x used in homePage

**Pull request merged**